graylog2-web-interface/src/components/common/Select.jsx

```diff
@@ -173,6 +204,10 @@ class Select extends React.Component<Props, State> {
     onChange(value);
   };
 
+  _onMenuOpen = () => this.setState({ isMenuOpen: true });
+
+  _onMenuClose = () => this.setState({ isMenuOpen: false });
```

Comment:
If we style the `DropdownIndicator` instead, we can just remove all this stuff.

Reply:
✅